modules/list.js

```diff
@@ -1,58 +1,58 @@
 export default class Books {
-    constructor() {
-        this.books = [];
-    }
+  constructor() {
+    this.books = [];
+  }
 
     addBook = (book) => {
-        this.books.push(book);
-        this.saveBooks();
-        this.renderBooks();
+      this.books.push(book);
+      this.saveBooks();
+      this.renderBooks();
     }
 
     removeBook = (id) => {
-        this.books = this.books.filter((book) => book.id !== Number(id));
-        this.saveBooks();
-        this.renderBooks();
-        let count = Number(localStorage.getItem('count'));
-        count -= 1;
-        localStorage.setItem('count', count);
+      this.books = this.books.filter((book) => book.id !== Number(id));
```

Comment:
* Nice Job so far 👏 just one little bug I noticed when I try to delete the first book added which doesn't respond below is a recording of the bug 👇 

https://user-images.githubusercontent.com/52497246/229973860-1fa0ea0d-c1df-48bf-a8f1-73137beeede7.mp4